styles/Home.module.css

```diff
@@ -74,6 +74,42 @@
   border: 1px solid rgba(var(--callout-border-rgb), 0.3);
   border-radius: var(--border-radius);
 }
+.statusBar{
+  width:100%;
+  height:20px;
+  border: 2px solid black;
+  border-radius: 5%;
+}
+#statusBarPoints{
+  width: 100%;
+  height:100%;
+  background-color: red;
+}
+
+#statusBarAssists{
```

Comment:
@devin-fitzsimons-kr Each of these are specific to its status bar.

Reply:
@devin-fitzsimons-kr I removed the heights in the status bar ids and placed it into a class. The widths are in the status bar ids because they represent the progress percentage.